photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java

```diff
@@ -174,7 +174,18 @@ public void setVideoModeInternal(VideoMode videoMode) {
                 videoModes = new HashMap<>();
                 List<VideoMode> videoModesList = new ArrayList<>();
                 try {
-                    var modes = camera.enumerateVideoModes();
+                    VideoMode[] modes;
+                    if (cameraQuirks.hasQuirk(CameraQuirk.PiCam)) {
+                        modes =
+                                new VideoMode[] {
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 320, 240, 90),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 960, 720, 15),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 1280, 720, 4),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 1920, 1080, 1),
```

Comment:
What's the deal with these framerates?

Reply:
These are in MMAL mode

Reply:
Is 640x480 not available in MMAL mode?

Reply:
Oh. We can do whatever we want. But I can add that